documentation/modules/adding-source-provider.adoc

```diff
@@ -78,10 +80,11 @@ ifdef::vmware[]
 . Specify the following fields:
 
 * *Provider resource name*: Name of the source provider.
+* *Endpoint type*: Select the vSphere provider endpoint type. Options: *vCenter* or *ESXi*.
```

Comment:
or maybe regarding the previous comment, this is the documentation of 2.6 so we can drop that statement above (or move it to the release notes) and add it here:
 ```suggestion
* *Endpoint type*: Select the vSphere provider endpoint type. Options: *vCenter* or *ESXi*. ESXi can be used for standalone servers, i.e.,  an ESX/ESXi server that is not managed by vCenter, or from ESX/ESXi servers that are managed by vCenter without going through vCenter.
```
what do you think?

Reply:
@ahadas , I made some changes based on your suggestions. Please review the PR again.